buildh/UI.py

```diff
@@ -68,9 +77,14 @@ def parse_cli():
     parser.add_argument("-t", "--traj", type=isfile,
                         help="Input trajectory file. Could be in XTC, TRR or DCD format.")
     parser.add_argument("-l", "--lipid", type=str, required=True,
-                        help="Residue name of lipid to calculate the OP on (e.g. POPC).")
+                        help="Combinaison of the name of the ForceField "
+                        "and the residue name of lipid to calculate the OP on (e.g. Berger_POPC)."
```

Comment:
Combination of ForceField name and residue name for the lipid to calculate the OP on (e.g. Berger_POPC)."